app/src/main/java/com/swayy/musicpark/presentation/NapsterApp.kt

```diff
@@ -2,6 +2,16 @@ package com.swayy.musicpark.presentation
 
 import android.app.Application
 import dagger.hilt.android.HiltAndroidApp
+import timber.log.Timber
 
 @HiltAndroidApp
-class NapsterApp: Application()
\ No newline at end of file
+class NapsterApp : Application() {
+    override fun onCreate() {
+        super.onCreate()
+        setupTimber()
+    }
+}
+
+private fun setupTimber() {
+    Timber.plant(Timber.DebugTree())
```

Comment:
👍 Consider checking if the build being run is debug to ensure logs are visible only for debug builds not release builds ---> if (BuildConfig.DEBUG) Timber.plant(Timber.DebugTree())

Reply:
sure ,I will customize timber for debug and release.

Reply:
is it okay now ? @Felix-Kariuki